warrior/Actions/GnmiActions/gnmi_actions.py

```diff
@@ -0,0 +1,362 @@
+'''
+Copyright 2017, Fujitsu Network Communications, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+
+"""
+gRPC Network Management Interface (gNMI) defines gRPC-based protocol for the modification and
+retrieval of configuration from a network element.
+As well as the control and generation of telemetry streams from a network element 
+to a data collection system.
+Warrior Keyword will support the gNMI Client Services.
+gNMI Services:
+1.	Capabilities:
+Exchange gNMI version and model info (model name, version, organization)
+2.	Get: (Subscribe once is Get)
+Separation of config and state
+3.	Set:
+Delete /replace/update operations supported
+4.	Subscribe
+Once/Polling/Stream
+"""
+
+import os, re, sys
+from Framework.ClassUtils.gnmi_utils_class import gNMI
+import Framework.Utils as Utils
+from Framework.Utils import testcase_Utils, config_Utils, data_Utils, file_Utils
+from time import sleep
+
+class gnmiactions(object):
+    """
+    Class for gNMI Action Keywords
+    """
+
+    def __init__(self):
+        self.resultfile = Utils.config_Utils.resultfile
+        self.datafile = Utils.config_Utils.datafile
+        self.logsdir = Utils.config_Utils.logsdir
+        self.filename = Utils.config_Utils.filename
+        self.logfile = Utils.config_Utils.logfile
+        self.tc_path = Utils.config_Utils.tc_path
+
+    def get_client_certificate(self, system_name, op_type="sftp", external_system=None,
+                               external_system_session=None):
+        """
+        Get client side certificate(.ca file) from gNMI server using SCP
+        and copy it at Warrior Config_files directory.
+        :param system_name:
+        :param op_type: What type of operation user want to perform "scp/sftp" default "sftp"
+        :return: True or False
+        """
+        wdesc = "Get client side certificate from gNMI server"
+        testcase_Utils.pSubStep(wdesc)
+        gnmi_execute = gNMI()
+        status = False
+
+        param = ['username', 'password', 'ip', 'gNMI_CLI_binary', 'sftp_port', 'gNMI_CA']

```

Comment:
what is gNMI_CA, how to use it is the data file.
Please update the documentation with details of what tags to be used in the datafile with example.
Also update the design document in the jira ticket with gNMI_CA details.

Reply:
Removing this keyword..
Earlier, there was only 1 certificate that was being used and it was getting created in NE. So, we needed a keyword to get the certificate. 
But now, we use openssl and get 3 certificates and directly use them. Hence, this keyword is not required now.